src/views/tutorial-view/index.tsx

```diff
@@ -1,41 +1,114 @@
+  FeaturedInCollections,
+  CollectionCardProps,
+  Badges,
+  getIsBeta,
+} from './components'
 
-export interface TutorialViewProps extends Omit<ClientTutorial, 'content'> {
+export interface TutorialViewProps
+  extends Pick<
+    ClientTutorial,
+    | 'name'
+    | 'slug'
+    | 'readTime'
+    | 'productsUsed'
+    | 'edition'
+    | 'handsOnLab'
+    | 'video'
+  > {
+  collectionCtx: CollectionContext
   content: MDXRemoteSerializeResult
+  layout: TutorialSidebarSidecarProps
 }
 
-// @TODO update this interface once we have a better idea of the page needs
+export type CollectionContext = {
+  isDefault: boolean
+  current: ClientCollection
+  featuredIn?: CollectionCardProps[]
+}
+
+export type TutorialSidebarSidecarProps = Required<
+  Pick<SidebarSidecarLayoutProps, 'children' | 'headings' | 'breadcrumbLinks'>
+>
+
+/**
+ *
+ * Outstanding @TODOs
+ * - add canonical url if this is the default collection
+ * - fix: the toc overview linking isn't properly aligning
+ * - wire up instruqt embed for interactive labs
+ * - skeleton out the next / prev component after API endpoint is updated - https://app.asana.com/0/1201903760348480/1201932088801131/f
+ */
+
 export default function TutorialView({
   name,
+  slug,
   content,
+  layout,
+  readTime,
+  productsUsed,
+  edition,
+  handsOnLab,
+  video,
+  collectionCtx,
 }: TutorialViewProps): React.ReactElement {
+  const { asPath } = useRouter()
   return (
     <SidebarSidecarLayout
+      breadcrumbLinks={layout.breadcrumbLinks}
       sidebarSlot={
-        <>
-          {new Array(20).fill(null).map((_) => (
-            <div key={_}>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
-              eros diam, fringilla ac malesuada vel, faucibus quis mauris.
-            </div>
-          ))}
-        </>
-      }
-      sidecarSlot={
-        <div>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur
-          eros diam, fringilla ac malesuada vel, faucibus quis mauris.
-        </div>
+        <Sidebar
+          title={collectionCtx.current.shortName}
+          menuItems={collectionCtx.current.tutorials.map((t) =>
+            formatTutorialToMenuItem(t, collectionCtx.current.slug, asPath)
+          )}
+        />
       }
+      headings={layout.headings}
     >
-      <h1>{name}</h1>
+      <header id="overview">
+        <Heading
+          level={1}
+          size={500}
+          weight="bold"
+          slug={layout.headings[0].slug}
+          id={slug}
+        >
+          {name}
+        </Heading>
+        <Badges
+          tutorialMeta={{
+            readTime,
+            products: productsUsed.map((p) => p.product.slug),
+            isBeta: getIsBeta(productsUsed),
+            edition,
+            hasVideo: Boolean(video),
+            isInteractive: Boolean(handsOnLab),
+          }}
+        />
+        {handsOnLab?.id ? <button>Show Terminal</button> : null}
+      </header>
       <Content
```

Comment:
Doesn't need to happen here, but the new `DevDotContent` component has been merged whenever you're ready to start using it. 😊

Reply:
Oh awesome, ya I'll probably leave that for another PR

Reply:
Sounds good to me!